### DISCUSSION TITLE
[1.20.1] Mob Imprisonment Tool Blacklist Config

### DISCUSSION THREAD

**Discussion by Reaviik** [InnovativeOnlineIndustries/Industrial-Foregoing#1427] [Feature Requests]
Make it possible to add entities to the blacklist. I'm doing this to block players from capturing NPCs and/or creatures that could be problematic.
![image](https://github.com/InnovativeOnlineIndustries/Industrial-Foregoing/assets/122496671/e0c15ccc-7815-44d0-be90-8545f3a01e77)

**Comment by Buuz135** [comment]
You can already do that by adding the entity to the the entity tag "industrialforegoing:mob_imprisonment_tool_blacklist"
